### DISCUSSION TITLE
Using .net custom templating can I change the path in config to folder structure in which it is created

### DISCUSSION THREAD

**Discussion by rahul-kanhaiya** [dotnet/templating#6233] [General]
I created a custom config and then generated a project from it, now my project folder structure is test/firstDraft , which should be updated in app settings file for property currDir. {currDir:'test/firstDraft'}

**Comment by YuliiaKovalova** [comment]
Hi @rahul-kanhaiya,

You can try to use _replace_ macro for updating currDir value. If I understand you correctly you can take a desired value from sourceName property. 
Please observe [examples]( https://github.com/dotnet/templating/blob/main/dotnet-template-samples/content/02-add-parameters/MyProject.Con/.template.config/template.json#:~:text=Smith%22%2C-,%22replaces%22%3A%22Contoso%22,-%7D%2C) in our repo.
